assignment/bundles/resources/assignment.properties

```diff
@@ -248,6 +248,7 @@ gen.thearecur2    = There are currently no submissions at this location.
 gen.theassall4    = This assignment allows submissions using both the text box below and attached documents. Type your submission in the box below and/or use the Browse button or the "select files" button to include other documents. <b>Save frequently while working</b>.
 gen.theassall6    = This assignment allows submissions using the text box below only. Type in your submission. <b>Save frequently while working</b>.
 gen.yoursubwill   = Your submission will be sent to {0} to be electronically reviewed for plagiarism.
+gen.yoursubwill.indexed = Your submission will be sent to {0} to be electronically reviewed for plagiarism. It will be indexed to be compared against by other submissions.
```

Comment:
I think "electronically" unnecessary these days ;)

It will be indexed to be compared against other submissions.

Reply:
Good call, I'll remove the 'electronically' from these two messages.
As for the indexing: when you submit to turnitin, it's always going to compare against other submissions; but when indexing is turned on, it allows other submissions to compare against the paper you're submitting

Reply:
sorry second comment was grammar: "against by other submissions"

Reply:
"against by other submissions" is correct. We always compare against other submissions; but the indexing setting allows other submissions to compare against this paper, I.e. this paper will be compared against by other submissions. Makes sense?